app/workers/alert_mailer_worker.rb

```diff
@@ -3,14 +3,14 @@
 class AlertMailerWorker
   include Sidekiq::Worker
 
-  def self.schedule_email(alert:, user:)
-    perform_async(alert.id, user.id)
+  def self.schedule_email(alert)
+    perform_async(alert.id, alert.target_user.id)
```

Comment:
The `self.schedule_email method` here, following the pattern I set up on other workers, is just to make the interface more explicit, so that you don't have to know the sidekiq limitations (or do the conversion of object to object id) when just using the worker.